src/proto/message_translation/tbots_protobuf.h

```diff
@@ -12,11 +12,13 @@
  * Returns a TbotsProto::World proto given a World.
  *
  * @param world The world msg to extract the TbotsProto::World from
+ * @param sequence_number An optional sequence number for tracking the TbotsProto::World
  *
  * @return The unique_ptr to a TbotsProto::World proto containing the field, friendly
  * team, enemy team, ball, and the game state.
  */
-std::unique_ptr<TbotsProto::World> createWorld(const World& world);
+std::unique_ptr<TbotsProto::World> createWorld(const World& world,
+                                               const uint64_t sequence_number = 0);
```

Comment:
it's probably clearer to just make two `createWorld(const World& world)` functions one with sequence number and one without